projects/com.oracle.truffle.llvm.parser.impl/src/com/oracle/truffle/llvm/parser/impl/LLVMVisitor.java

```diff
@@ -1164,7 +1164,8 @@ private LLVMExpressionNode getConstantElementPtr(LLVMExpressionNode baseAddress,
         }
         if (currentOffset != 0) {
             LLVMExpressionNode oneValueNode = factoryFacade.createLiteral(1, LLVMBaseType.I32);
```

Comment:
Can we instead of introducing a new field in the `NodeFactoryFacade` just adapt the value of the `oneValueNode` to be `currentOffset/stride`?